github_dynamic_targets/impl.py

```diff
@@ -9,3 +11,40 @@ def get_approvers(event):
 
     # allow_self lets the requester approve themself, which is great for testing!
     return slack.channel("#sym-requests", allow_self=True)
+
+
+# Prefetch reducers return a list of FieldOptions that will be used to populate
+# the Slack select menu for the Prompt Field with the given "field_name"
+@prefetch(field_name="repo_name")
+def get_repos(event):
+    github_token = event.flow.environment.integrations["github"].settings["api_token_secret"].retrieve_value()
+    headers = {"Authorization": f"Token {github_token}"}
+
+    all_options = []
+    per_page = 10
```

Comment:
should we make this bigger so that we have to make less requests?

Reply:
Actually there's only 3 repos in the Sym Test organization l o l 😂

Reply:
oh lol ok